tensorflow/python/kernel_tests/number_to_string_op_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for NumberToString op from parsing_ops."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow.python.framework import dtypes
+from tensorflow.python.framework import test_util
+from tensorflow.python.ops import array_ops
+from tensorflow.python.ops import parsing_ops
+from tensorflow.python.platform import test
+
+
+class NumberToStringOpTest(test.TestCase):
+
+  def _test(self, tf_type, test_cases):
+    with self.cached_session():
+      # Build a small testing graph.
+      input_value = array_ops.placeholder(tf_type)
```

Comment:
This is great. Please also include a test or two with the following inputs:

`constant_op.constant`
`ragged_tensor.RaggedTensor` (from tensorflow.python.ops.ragged import ragged_tensor)

We also want to see what happens for non-numeric inputs, for example a `Variable` and a `TensorArray` (we expect an error).